frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -195,6 +197,10 @@ const DataTable = ({
     }
   }, [selectedDropdownFilter]);
 
+  useEffect(() => {
+    toggleAllRowsSelected(false);
+  }, [clearSelectionCount]);
+
```

Comment:
This an approach for calling a child component's function without breaking React's one way data flow.